build/rpmbuild-docker/Dockerfile

```diff
@@ -23,16 +23,23 @@ RUN yum clean all && rm -rf /var/cache/yum
 
 ENV GO_VERSION 1.14.4
```

Comment:
https://github.com/Percona-Lab/pmm-submodules/pull/1079

Reply:
Sure, it'll be rebased

Reply:
@AlekSi @EvgeniyPatlan Rebased in 23e70a5